asmcomp/arm64/emit.mlp

```diff
@@ -745,9 +745,9 @@ let emit_instr env i =
           `	bl	{emit_symbol "caml_c_call"}\n`;
           `{record_frame env i.live (Dbg_other i.dbg)}\n`
         end else begin
-          (* store ocaml stack in the frame pointer register
-             NB: no need to store previous x29 because OCaml frames don't
-             maintain frame pointer *)
+          (* Push frame pointer (x29) onto stack and restore later. *)
+          `	stp	x29, x30, [sp, -16]!\n`;
```

Comment:
Is there a need to save x30 here? You could simply do
```suggestion
          `	str	x29, [sp, -16]!\n`;
```
which preserves the stack alignment.

Reply:
No there really isn't. Just storing `x29` is sufficient.